SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -157,7 +157,7 @@ bool USpatialPackageMapClient::ResolveEntityActor(AActor* Actor, Worker_EntityId
 
 	if (GetEntityIdFromObject(Actor) == SpatialConstants::INVALID_ENTITY_ID)
```

Comment:
driveby - should this be `!= EntityId` ?

Reply:
don't see why not
changed